lib/cocina/models/builders/title_builder.rb

```diff
@@ -133,14 +140,16 @@ def other_title(titles)
         # @return [String] the title value from combining the pieces of the structured_values by type and order
         #   with desired punctuation per specs
         #
-        # rubocop:disable Metrics/BlockLength
+        # rubocop:disable Metrics/AbcSize
         # rubocop:disable Metrics/CyclomaticComplexity
         # rubocop:disable Metrics/MethodLength
         # rubocop:disable Metrics/PerceivedComplexity
         def title_from_structured_values(title)
-          result = ''
+          # parse out the parts
+          main_title = ''
+          subtitle = ''
+          non_sort_value = ''
```

Comment:
the changes to this method appear super ugly unless you use split view.   Note the comments for "parse out the parts" on line 148, and "combine the parts" on line 185.